src/hydrox/core/patch.clj

```diff
@@ -9,7 +9,7 @@
   [reader initch opts]
   (let [ch (reader/read-char reader)]
     (if-not (utils/whitespace? ch)
-      (let [token (#'edn/read-token reader ch)]
+      (let [token (#'edn/read-token reader "kind" ch)]
```

Comment:
Perhaps this should be `"keyword"` since that is the `kind` to be reported on error.

Reply:
I'd like to somehow propagate that through instead of putting a placeholder in but I was finding it hard to figure out where that happens. Do you know?

Reply:
https://github.com/parkside-securities/hydrox/blob/master/src/hydrox/core.clj#L89

Reply:
I'm trying to see why this patch was introduced in the first place. 

Here is the current version: https://github.com/clojure/tools.reader/blob/master/src/main/clojure/clojure/tools/reader/edn.clj#L265-L278

Reply:
Yeah, I found both of these things, which is how I was able to fix the function to make it compatible in the first place. I was looking for the rationale behind the patch, and how to use it to give the better error messages promised by the change that introduced `kind` to the reader, the PR for which I have gone over but was not particularly illuminating.

Reply:
https://github.com/helpshift/hydrox/commit/f9dea2c1dac9d1ecfb8ea9b2c420f1fe4f628028 introduced the change upstream but unfortunately, I am not seeing anything descriptive in the commit messages.

Reply:
I think replacing `"kind"` with `:keyword` as [they do here](https://github.com/clojure/tools.reader/blob/master/src/main/clojure/clojure/tools/reader/edn.clj#L269) would be sufficient.